src/binance/bnbEngine.ts

```diff
@@ -400,17 +394,16 @@ export class BinanceEngine extends CurrencyEngine<BinancePlugin> {
     const data =
       spendTarget.otherParams != null ? spendTarget.otherParams.data : undefined
 
-    let otherParams: Object = {}
+    let otherParams: BinanceTxOtherParams | undefined
```

Comment:
I believe this just needs to be BinanceTxOtherParams since it gets defined no matter what. That alone doesn't really matter BUT maybe removing the conditional below it is because we only support BNB transfers and the else condition is arguably dead code.